src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx

```diff
@@ -209,11 +209,13 @@ export const EVMDestinationAdaptorProvider = ({
     const fallback = new Fallback(delayMs, pollingIntervalMs, async () => {
       let res;
       try {
-        res = await destinationBridge?.getProposal(
-          srcChainId,
-          parseInt(depositNonce as string),
-          erc20ProposalHash
-        );
+        if(depositNonce) {
```

Comment:
How does it relate to `polkadot/api` lib update?

Reply:
parseInt(depositNonce) of undefined returns NaN.  Using NaN  return error in the new polkadot.js library.